apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/FetchPolicyInterceptors.kt

```diff
@@ -72,26 +74,25 @@ val CacheFirstInterceptor = object : ApolloInterceptor {
         } else {
           throw it
         }
-      }.singleOrNull()
+      }.map { response ->
```

Comment:
This breaks the contract that `CacheFirst` [emits only once](https://github.com/apollographql/apollo-kotlin/blob/2e21bd2952d75e86c58920c498568ab73d632774/apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/ClientCacheExtensions.kt#L40). I guess there's no real way around this but the doc needs to be updated